src/config/dynamic-theme-fixes.config

```diff
@@ -25839,6 +25839,16 @@ IGNORE INLINE STYLE
 
 ================================
 
+www.webofscience.com
+
+CSS
+.hh, .labl, mark{
+    color: var(--darkreader-neutral-text) !important;
+    background-color: ${COLOR} !important;
```

Comment:
Please replace `COLOR` with the actual source color (before the inversion), like white or black, or use `var(--darkreader-neutral-background)`

Reply:
Thanks, changed color to gray. Any white-black, or user var do not provide the correct contrast. Please refer to images below where the search text is "artificial intelligence"

![image](https://github.com/darkreader/darkreader/assets/110386035/a2eb192e-2db9-481f-bd2c-0d22c13b702a)

![image](https://github.com/darkreader/darkreader/assets/110386035/813a8fe0-4b5a-4f6c-ad77-4de8e406f01a)